rules/no-new-symbol.yml

```diff
@@ -0,0 +1,6 @@
+id: no-new-symbol
+message: Symbol cannot be called as a constructor.
+severity: error
+language: TypeScript
+rule:
+  pattern: new Symbol($$$)
```

Comment:
Can we also include the `note` field in the rule? Copy the text from eslint official site is good.

https://eslint.org/docs/latest/rules/no-new-symbol
```
Symbol is not intended to be used with the new operator, but to be called as a function.
```

Reply:
😮 I forgot to include it. Added it.